cookbooks/continuous_delivery.rst

```diff
@@ -0,0 +1,187 @@
+have to generate an SSH key, and register it in SymfonyCloud:
+
+.. code-block:: bash
+
+   ssh-keygen -N '' -f symfonycloud-ci.key SYMFONY_TOKEN=xxxx symfony
+   account:ssh:key:add symfonycloud-ci.key
+
+.. caution::
+
+   Don't forget to register and expose the ``SYMFONY_TOKEN`` environment value
+   as well as the generated SSH key in your automation system.
+
+.. _cd-env-flags:
+
+CLI flags and environment variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The CLI tool defines several flags and environment flags that allow to change
+it's behavior. As a general rule, for automated context, environment variables
+are preferable but flags can be used ponctually if this is required.
+
+SYMFONY_PROJECT
+"""""""""""""""
+
+Setting this environment value make useless to run ``symfony project:link``. Its
+value should be set to the SymfonyCloud project's id that can be found with
+``symfony project:info`` or in the emails sent by SymfonyCloud.
+
+.. tip:: One can also use the ``--project`` flag.
+
+SYMFONY_ENVIRONMENT
+"""""""""""""""""""
+
+By default, when SymfonyCloud needs to work on the remote environment, it tries
+to find an exising one whose name matches the current Git branch name. One might
+want to change this logic, this can be done by setting this environment variable
+with a different value. You can use the ``symfony envs`` commands to list
+environments currently present on SymfonyCloud.
+
+.. tip:: One can also use the ``--environment`` flag.
+
+.. _cd-env-create-delete:
+
+Creating and deleting environments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``symfony env:create <name> <parent>`` command creates a new SymfonyCloud
+environment. Most users don't need to provide any argument when running it
+locally, but when running inside a script it is best to specify them. The first
+argument is the name desired for the environment to be created on SymfonyCloud.
+The second argument is the name of the "parent" environment. The new environment
+will be created first as a copy of the parent and as result inherits the
+parent's code and data.
+
+Deleting an environment can be made using the ``symfony project env:delete
+<name>`` command.
+
+.. _cd-url:
+
+Environments URLs
+^^^^^^^^^^^^^^^^^
+
+Each SymfonyCloud environment gets a unique subdomain that can not be determined
+before its creation. Once created this domain is stable.
+
+This implies that the application code has to consider this: avoid any
+hard-coded domain values. Even Symfony parameters should be avoided and the use
+of the current request's host should be prefered. This might also make working
+with systems like OAuth, SSO or webhooks difficult. To help you work with this,
+SymfonyCloud exposes a ``SYMFONY_PROJECT_DEFAULT_ROUTE_URL`` environment
+variable, more information in `the dedicated section
+<symfonycloud-environment-variables>`_.
+
+To determine the available URLs from your automated system, please use the
```

Comment:
I can also run the command locally, so this is not limited to automated systems.

Reply:
There's no mention of such limitation.